screwdriver.yaml

```diff
@@ -31,6 +31,8 @@ jobs:
     requires:
       - ~pr
       - ~commit
+      - ~commit:0.2.x-alpha
+      - ~pr:0.2.x-alpha
```

Comment:
We should probably add a `main-0.2.x-alpha` build (just `npx lerna run test`) since this one allows reports/dashboards to fail